docs/tutorial.rst

```diff
@@ -1,4 +1,4 @@
-.. _aiohttp-tutorial:
+.. _tutorial-beginning:
```

Comment:
Think about crosslinks. Better to use prefix like aiohttp-demos- everywhere in demos documentation.

Reply:
makes sense